simplified-app-openebooks/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "83753990150",
+    "firebase_url": "https://open-ebooks-nypl.firebaseio.com",
+    "project_id": "open-ebooks-nypl",
+    "storage_bucket": "open-ebooks-nypl.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:83753990150:android:770193bd9ffeb14e83eb87",
+        "android_client_info": {
+          "package_name": "org.nypl.labs.OpenEbooks.app"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "83753990150-3artgccg30u1svjr45a97embu57ut8cf.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyBzGt9lNXa3vO0zJccs_qlPQrwDhEYCR6c"
```

Comment:
Is this really not a secret?

Reply:
Yep! You can grab any published APK file and extract this key with no special tools. Any web or mobile app using the Firebase API will have an API key like this.

Google is pretty explicit here (emphasis mine):

> The content of the Firebase config file or object **_is considered public_**, including the app's platform-specific ID (iOS bundle ID or Android package name) and the Firebase project-specific values, like the API Key, project ID, Realtime Database URL, and Storage bucket name.

Source https://firebase.google.com/docs/projects/learn-more#config-files-objects

See also [here](https://groups.google.com/g/firebase-talk/c/bamCgTDajkw/m/uVEJXjtiBwAJ?pli=1) and [here](https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-api).